--- jobOptions_tagdl1.py	2016-05-17 11:18:21.000000001 +0200
+++ ../../run/jobOptions_ipmp.py	2016-05-17 16:22:54.000000001 +0200
@@ -2,7 +2,7 @@
 ##########################################################################################################################################################
 ### MAIN SWITCHES
 
-ONLYEssentialInfo =False   ## write minimal amount of info on the output file
+ONLYEssentialInfo =True   ## write minimal amount of info on the output file
 ReduceInfo        =False    ## write minimal amount of info on the output file
 DoMSV             =True   ## include variables for MSV tagger
 doSMT             =False   ## include variables for SMT tagger
@@ -140,7 +140,7 @@
 from BTagging.BTaggingFlags import BTaggingFlags
 
 #### if the new file is already in the datatbase: simple edit the name
-###############BTaggingFlags.CalibrationTag = 'BTagCalibRUN12-08-10'
+BTaggingFlags.CalibrationTag = 'BTagCalibRUN12-08-18'
 
 #### if you want to use your own calibration file use this part below
 #BTaggingFlags.CalibrationFromLocalReplica = True
@@ -149,12 +149,17 @@
 
 # if the calibration for DL1 is to be read in from a local JSON file
 BTaggingFlags.DL1 = True
-BTaggingFlags.DL1LocalNNConfig = "BTagging_DL1_NNconfig.json"
+# BTaggingFlags.DL1LocalNNConfig = "BTagging_DL1_NNconfig.json"
 
-BTaggingFlags.TagNtupleDumper = True
-BTaggingFlags.TagNtupleStream = "FTAG"
-svcMgr.THistSvc.Output.append(
-  "FTAG DATAFILE='tagger_inputs.root' OPT='RECREATE'")
+# BTaggingFlags.TagNtupleDumper = True
+# BTaggingFlags.TagNtupleStream = "FTAG"
+# svcMgr.THistSvc.Output.append(
+#   "FTAG DATAFILE='tagger_inputs.root' OPT='RECREATE'")
+
+# blank second field means read from file
+BTaggingFlags.RNNIP = True
+BTaggingFlags.RNNIPConfig = {'ipmp': 'ipmp.json', 'ipmk': 'ipmk.json'}
+BTaggingFlags.OutputLevel = DEBUG
 
 include("RetagFragment.py")
 if doRetag:
@@ -205,6 +210,13 @@
 ##########################################################################################################################################################
 ##########################################################################################################################################################
 
+ipnn_inputs = [
+  "d0", "d0sig", "z0", "z0sig", "d0z0sig", "grade", "fromV0", "pt", "dPhi",
+  "absEta", "chi2", "nInnHits", "nNextToInnHits", "nBLHits", "nsharedBLHits",
+  "nsplitBLHits", "nPixHits", "nsharedPixHits", "nsplitPixHits", "nSCTHits",
+  "nsharedSCTHits", "expectBLayerHit",
+]
+
 ### Main Ntuple Dumper Algorithm
 for JetCollection in JetCollections:
   shortJetName=JetCollection.replace("AntiKt","Akt").replace("TopoJets","To").replace("TrackJets","Tr")
@@ -237,6 +249,21 @@
   alg.JetCleaningTool.DoUgly  = True
   if not doComputeReference: algSeq += alg
 
+  # add RNN outputs
+  for tagger in ['ipmp']:
+    for flav in ['pu', 'pc', 'pb', 'ptau']:
+      alg.ArbitraryDoubleBranches.append(tagger + '_' + flav)
+    for inpt in ipnn_inputs:
+      alg.ArbitraryFloatVectorBranches.append(tagger + '_' + inpt)
+
+  # SLAC version
+  alg.ArbitraryDoubleBranches.append('ipmk_pb')
+
+  # DL1 outputs (only use if we have "essential info" turned on)
+  if ONLYEssentialInfo:
+    for flav in 'bcu':
+      alg.ArbitraryDoubleBranches.append('DL1_p' + flav)
+
   ###print JetCollection
   calibfile        = "JES_Prerecommendation2015_Feb2015.config"
   collectionForTool="AntiKt4LCTopo"
