--- jobOptions_tagdl1.py	2016-05-15 17:34:32.000000001 +0200
+++ ../../run/jobOptions_tagdl1.py	2016-05-15 16:20:51.000000001 +0200
@@ -2,7 +2,7 @@
 ##########################################################################################################################################################
 ### MAIN SWITCHES
 
-ONLYEssentialInfo =False   ## write minimal amount of info on the output file
+ONLYEssentialInfo =True   ## write minimal amount of info on the output file
 ReduceInfo        =False    ## write minimal amount of info on the output file
 DoMSV             =True   ## include variables for MSV tagger
 doSMT             =False   ## include variables for SMT tagger
@@ -148,13 +148,18 @@
 #BTaggingFlags.CalibrationTag = 'BTagCalibRUN2-test'
 
 # if the calibration for DL1 is to be read in from a local JSON file
-BTaggingFlags.DL1 = True
-BTaggingFlags.DL1LocalNNConfig = "BTagging_DL1_NNconfig.json"
+# BTaggingFlags.DL1 = True
+# BTaggingFlags.DL1LocalNNConfig = "BTagging_DL1_NNconfig.json"
 
-BTaggingFlags.TagNtupleDumper = True
-BTaggingFlags.TagNtupleStream = "FTAG"
-svcMgr.THistSvc.Output.append(
-  "FTAG DATAFILE='tagger_inputs.root' OPT='RECREATE'")
+# BTaggingFlags.TagNtupleDumper = True
+# BTaggingFlags.TagNtupleStream = "FTAG"
+# svcMgr.THistSvc.Output.append(
+#   "FTAG DATAFILE='tagger_inputs.root' OPT='RECREATE'")
+
+# blank second field means read from file
+BTaggingFlags.RNNIP = True
+BTaggingFlags.RNNIPConfig = {'ipmp': 'ipmp.json'}
+BTaggingFlags.OutputLevel = INFO
 
 include("RetagFragment.py")
 if doRetag:
@@ -237,6 +242,12 @@
   alg.JetCleaningTool.DoUgly  = True
   if not doComputeReference: algSeq += alg
 
+  # add RNN outputs
+  for tagger in BTaggingFlags.RNNIPConfig:
+    for flav in ['pu', 'pc', 'pb', 'ptau']:
+      alg.ArbitraryBranchNames.append(tagger + '_' + flav)
+
+
   ###print JetCollection
   calibfile        = "JES_Prerecommendation2015_Feb2015.config"
   collectionForTool="AntiKt4LCTopo"
