Index: jobOptions_tagdl1.py
===================================================================
--- jobOptions_tagdl1.py	(revision 321829)
+++ jobOptions_tagdl1.py	(working copy)
@@ -156,6 +156,11 @@
 svcMgr.THistSvc.Output.append(
   "FTAG DATAFILE='tagger_inputs.root' OPT='RECREATE'")
 
+# blank second field means read from file
+BTaggingFlags.RNNIP = True
+BTaggingFlags.RNNIPConfig = {'ipmp': 'ipmp.json'}
+BTaggingFlags.OutputLevel = INFO
+
 include("RetagFragment.py")
 if doRetag:
   from BTagging.BTaggingConfiguration import getConfiguration
@@ -237,6 +242,12 @@
   alg.JetCleaningTool.DoUgly  = True
   if not doComputeReference: algSeq += alg
 
+  # add RNN outputs
+  for tagger in BTaggingFlags.RNNIPConfig:
+    for flav in ['pu', 'pc', 'pb', 'ptau']:
+      alg.ArbitraryBranchNames.append(tagger + '_' + flav)
+
+
   ###print JetCollection
   calibfile        = "JES_Prerecommendation2015_Feb2015.config"
   collectionForTool="AntiKt4LCTopo"
